src/components/Nav/Mobile/LvlAccordion.tsx

```diff
@@ -82,7 +82,7 @@ const LvlAccordion = ({
                     {label}
                   </Text>
                   <Text
-                    fontWeight="regular"
+                    fontWeight="normal"
```

Comment:
This did need correction, but the issue in question is coming from the [other return render farther down the component](https://github.com/ethereum/ethereum-org-website/blob/6ecf1d2d53f8e21ac56bb6d29870123ec517acb6/src/components/Nav/Mobile/LvlAccordion.tsx#L140) that also needed the correct value for `fontWeight`.

Currently there is no change to the text weight.

Reply:
@sanjanaynvsdl Agreed, keep in mind there are two returns here, conditioned on whether it's an actual link, or a submenu with deeper items. The one that was problematic in @konopkja's post was the submenu, which is on Line 140. 

All that being said, I'm actually not even able to reproduce this bug in production right now on a few devices. Also, [this file is currently being revamped via migration to tailwind](https://github.com/ethereum/ethereum-org-website/pull/13449/files#diff-5d04d66ce0feb057f40ccd79221664493d4019bb0a7295c144e9059e652aef53) in a simultaneous PR.

I can merge this current fix in, but I don't think it's worth more effort than that since the above PR is likely to be in by next deploy.